lib/modules/activity/reply_card.dart

```diff
@@ -146,6 +156,45 @@ class ReplyCard extends StatelessWidget {
   }
 }
 
+class _ReplyMentionButton extends StatefulWidget {
```

Comment:
I think this doesn't need to be a `StatefulWidget`.

Reply:
You're right, I tried this earlier and got an error, but it was because of an unrelated issue